scrapy/crawler.py

```diff
@@ -123,22 +127,21 @@ def __init__(self, settings):
         super(CrawlerProcess, self).__init__(settings)
         install_shutdown_handlers(self._signal_shutdown)
         self.stopping = False
-        self.log_observer = log.start_from_settings(self.settings)
-        log.scrapy_info(settings)
+        configure_logging(settings)
```

Comment:
Python [docs](https://docs.python.org/2/howto/logging.html#configuring-logging-for-a-library) suggest not to add logger handlers by default in libraries. Maybe we can move this from Crawler to Scrapy command-line module?